### ISSUE TITLE
NTR: 7SK RNA terms

### ISSUE THREAD

**Issue by sjm41** [The-Sequence-Ontology/SO-Ontologies#512]
**What is this request referring to?**
i) 7SK RNA gene
ii) 7SK RNA

**What is the name you would like SO to give the term?**
i) 7SK_RNA_gene
ii) 7SK_RNA

**What is the definition that you would like for this term?**
i) A gene encoding a 7SK RNA.
ii) An abundant small nuclear RNA that, together with associated cellular proteins, regulates the activity of the positive transcription elongation factor b (P-TEFb). (PMID: 21853533)

**Parent Term**
i) ncRNA_gene (SO:0001263)
ii) ncRNA (SO:0000655)
(note - 7SK RNA is often described as "a snRNA" but at 300+ nt, but at 400+nt, it's over the canonical definition of snRNA (<200nt).)

**Relevant Publications**
https://pubmed.ncbi.nlm.nih.gov/21853533/
https://pubmed.ncbi.nlm.nih.gov/19246988/
https://pubmed.ncbi.nlm.nih.gov/27369380/

**Comment by davidwsant** [comment]
Hi @sjm41,

Two terms have been added. I changed the names slightly to reflect SO formatting.

id: SO:0002340
name: RNA_7SK
def: "An abundant small nuclear RNA that, together with associated cellular proteins, regulates the activity of the positive transcription elongation factor b (P-TEFb). It is often described in literature as similar to a snRNA, except of longer length." [PMID:19246988, PMID:21853533, PMID:27369380]
synonym: "7SK RNA" EXACT []
synonym: "RNA 7SK" EXACT []
is_a: SO:0000655 ! ncRNA

id: SO:0002341
name: RNA_7SK_gene
def: "A gene encoding a 7SK RNA (SO:0002340)." [PMID:19246988, PMID:21853533, PMID:27369380]
synonym: "7SK RNA gene" EXACT []
synonym: "RNA 7SK gene" EXACT []
is_a: SO:0001263 ! ncRNA_gene
